packages/shylu/shylu_dd/frosch/src/SchwarzOperators/FROSch_CoarseOperator_def.hpp

```diff
@@ -664,6 +734,158 @@ namespace FROSch {
                 }
 #endif
             }
+        } else if(!DistributionList_->get("Type","linear").compare("ZoltanDual")){
+          //ZoltanDual provides a factorization of the coarse problem with Zoltan2 inlcuding the
```

Comment:
Does it provide a factorization or a partition?

Reply:
It is a partition. I will change that.